infrastructure/main.tf

```diff
@@ -170,7 +175,7 @@ resource "aws_cloudwatch_log_group" "example" {
 ##### CloudFront Distribution (Prod) #####
 ##------------------------------------####
 locals {
-    s3_origin_id = "cdn-s3-origin"
+  s3_origin_id = aws_s3_bucket.this.id
```

Comment:
@haicasgox @donglu1000tu @hamz21 
Nist, I think we should change this and also bucket name should more flexible
So with that case, random pet is not really perfect, however still ok for a demo project. Wdyt?

Reply:
LGTM, thanks for the formatting, please let see what others think as well.

Reply:
@hungran :  LGTM! Thank you for your PR.